src/utils/modals.tsx

```diff
@@ -0,0 +1,51 @@
+import { Text } from '@mantine/core';
+import { modals } from '@mantine/modals';
+import { RiErrorWarningFill } from 'react-icons/ri';
+
+import type { CompactJob } from '@/types/job';
+
+export function enableModal(clusterName: string, setEnabled: (enabled: boolean) => void) {
```

Comment:
Oh, it's a mistake. I was first trying to make a single generic modal component. But later realised that it will need customised text, title, different parameters as props for various uses. So decided to make different modals for different use cases, but forgot to move it to cluster's folder.